src/pages/Home.tsx

```diff
@@ -4,12 +4,20 @@ import TopNav from "../components/TopNav";
 
 // import MUI components
 import Container from "@mui/material/Container";
+<<<<<<< HEAD
+import Footer from "../components/Footer";
+=======
+>>>>>>> development
```

Comment:
The merge text needs to be deleted to resolve merge conflicts. <<<<<< HEAD, ======== and >>>>>>> development should be deleted. Typically you would also delete some conflicting code, but there isn't any conflicting code to delete here.